boards/nrf51dk/src/main.rs

```diff
@@ -210,24 +209,24 @@ pub unsafe fn reset_handler() {
     while !nrf51::clock::CLOCK.low_started() {}
     while !nrf51::clock::CLOCK.high_started() {}
 
-    let platform = static_init!(
-        Platform,
-        Platform {
-            gpio: gpio,
-            timer: timer,
-            console: console,
-            led: led,
-            button: button,
-        },
-        160/8);
+    let platform = Platform {
+        gpio: gpio,
+        timer: timer,
+        console: console,
+        led: led,
+        button: button,
```

Comment:
Because the nrf51dk doesn't have an MPU and doesn't support multiple processes, so IPC doesn't make sense